components/badge/overview.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Overview
+page_title: Badge Overview
+description: Discover the Badge component for Blazor. Learn how to add the component to your app and explore its configuration options, such as positioning and alignment.
+slug: badge-overview
+tags: telerik,blazor,badge,navbar
+published: True
+position: 0
+---
+
+# Blazor Badge Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/badge" target="_blank">Blazor Badge component</a> is a visual element that displays additional information, such as status indicators, notification icons, short text, and others.
+
+The Badge comes with built-in customization features that allow the developer to change the [background color]({%slug badge-appearance%}#themecolor) and the [fill mode]({%slug badge-appearance%}#fillmode), to enable or disable the cutout border, and fine-tune the [positioning and alignment]({%slug badge-position-alignment%}) of the component.
```

Comment:
Sort the features by importance. Positioning and alignment have a higher value than appearance settings.